commit 19bb00d596ac6893905b9df1d7e6f92e4d0ce186
Author: q66 <q66@chimera-linux.org>
Date:   Mon Jul 7 22:37:54 2025 +0200

    fix ABI corruption for on_tags_activated
    
    The callback was taking wrong arguments, as the callback is
    supposed to take two arguments, not three, and the user_data
    is in the second one. The third one was NULL presumably because
    the register was zeroed.
    
    This fixes crash when editing bookmark tags.

diff --git a/src/bookmarks/ephy-bookmark-properties.c b/src/bookmarks/ephy-bookmark-properties.c
index a981af4..d116d60 100644
--- a/src/bookmarks/ephy-bookmark-properties.c
+++ b/src/bookmarks/ephy-bookmark-properties.c
@@ -293,8 +293,7 @@ ephy_bookmark_properties_bookmark_tag_removed_cb (EphyBookmarkProperties *self,
 }
 
 static void
-on_tags_activated (GtkListBox    *box,
-                   GtkListBoxRow *row,
+on_tags_activated (AdwActionRow  *row,
                    gpointer       user_data)
 {
   EphyBookmarkProperties *self = EPHY_BOOKMARK_PROPERTIES (user_data);
